### PR TITLE
Added POST functionality that posts files from local to server/static

### PR DESCRIPTION
**Pull request by rrrliu**
fk yeah

### CODE REVIEW

File: client/src/components/Lessons/Modal/AddLessonModal.jsx [hunk 1]
```diff
@@ -43,6 +48,10 @@ class AddLessonModal extends Component {
     if (!lessonName) {
       alert('Please enter a lesson name.');
     } else {
+      const { currfile } = this.state;
+      const data = new FormData();
+      data.append('sampleFile', currfile);
```
Comment (partyshah) [inline client/src/components/Lessons/Modal/AddLessonModal.jsx]: make 'sampleFile' into a variable pls

File: client/src/components/Lessons/Modal/AddLessonModal.jsx [hunk 2]
```diff
@@ -69,6 +78,7 @@ class AddLessonModal extends Component {
             onDrop={this.onDrop}
             onDropRejected={onDropRejected}
             multiple={false}
+            inputProps={{ name: 'sampleFile' }}
```
Comment (partyshah) [inline client/src/components/Lessons/Modal/AddLessonModal.jsx]: again

File: server/routes/index.js [hunk 3]
```diff
@@ -111,4 +110,25 @@ router.get('/studentSummary/:unitID', async (req, res) => {
   }
 });
 
+router.post('/upload', async (req, res) => {
+  // if (Object.keys(req.body).length === 0) {
+  //   return res.status(400).send('No files were uploaded.');
+  // }
+  console.log('lmao');
```
Comment (partyshah) [inline server/routes/index.js]: what the actual fuck is wrong with you

File: client/src/components/Lessons/Modal/AddLessonModal.jsx [hunk 4]
```diff
@@ -43,6 +48,10 @@ class AddLessonModal extends Component {
     if (!lessonName) {
```
Comment (partyshah) [inline client/src/components/Lessons/Modal/AddLessonModal.jsx]: check if a fie exists

File: client/src/components/Lessons/Modal/AddLessonModal.jsx (hunk 1 above)
Comment (rrrliu) [reply client/src/components/Lessons/Modal/AddLessonModal.jsx]: done

File: server/routes/index.js (hunk 3 above)
Comment (rrrliu) [reply server/routes/index.js]: fuck you

File: client/src/components/Lessons/Modal/AddLessonModal.jsx (hunk 4 above)
Comment (rrrliu) [reply client/src/components/Lessons/Modal/AddLessonModal.jsx]: yee

File: client/src/components/Lessons/Modal/AddLessonModal.jsx (hunk 2 above)
Comment (rrrliu) [reply client/src/components/Lessons/Modal/AddLessonModal.jsx]: done

**Pull request merged**